src/Thumb.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div :class="['thumb-wrapper', addClass, {'thumb-circle': isCircle}]" :style="[getBorder, getFontSize, getFontColor, getBgColor]">
+    <img ref="pic" @load.once="computeImgSize" :src="src" class="thumb-image" :width="computedWidth" :height="computedHeight" v-if="isImage">
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+  const DEFAULT_SIZE = 100;
+
+  export default {
+    props: {
```

Comment:
Sort your props alphabetically, and by type. Although some other Vue components do not follow this, it is a good practice to start enforcing now (before migration to Bootstrap-Vue).

Reply:
fixed in latest push